File: Python/MongoDB/CRUD/mongodb_config.py
class MongoDBConfig:
    def __init__(self, username, password, server, retryWrites, w, appName):
        self.username = username
        self.password = password
        self.server = server
        self.retryWrites = retryWrites
        self.w = w
        self.appName = appName

    def to_uri(self) -> str:
        return f"mongodb+srv://{self.username}:{self.password}@{self.server}/?retryWrites={self.retryWrites}&w={self.w}&appName={self.appName}"

File: Examples/PHP/Containers/Laravel-MongoDB-REST/app/Services/MongoDB/CRUD/MongoDB_Config.php
<?php

namespace App\Services\MongoDB\CRUD;

class MongoDB_Config {
    private $username;
    private $password;
    private $server;
    private $retryWrites;
    private $w;
    private $appName;

    public function __construct($username, $password, $server, $retryWrites, $w, $appName) {
        $this->username = $username;
        $this->password = $password;
        $this->server = $server;
        $this->retryWrites = $retryWrites;
        $this->w = $w;
        $this->appName = $appName;
    }

    public function toUri() {
        return "mongodb+srv://{$this->username}:{$this->password}@{$this->server}/?retryWrites={$this->retryWrites}&w={$this->w}&appName={$this->appName}";
    }
}